docs/software/koji-workflow.md

```diff
@@ -47,53 +47,137 @@ The results of a buildArch task. Their metadata are recorded in the Koji databas
 **Repository**  
 A yum repository created from the contents of a tag at a specific point in time. By default, the yum repository will contain all successful, non-blocked builds in the tag, plus all RPMs in the external repositories for the tag.
 
-Using Koji
-----------
 
-### Required Software
+Obtaining Access
+----------------
+
+Building OSG packages in Koji requires these privileges:
+
+- access to the OSG subversion repository at https://vdt.cs.wisc.edu/svn
+- access to a login node at UW Comp Sci such as `moria.cs.wisc.edu`
+- access to the Koji service via a grid user certificate
+
+See the [user certificates document](https://opensciencegrid.org/docs/security/user-certs/)
+for information about how to get a user certificate.
+
+Open a Freshdesk ticket with the subject "Requesting access to Koji" with the following information:
+- top 3 username choices for the login node and SVN
+  (8 characters max, no punctuation)
+- the DN of your user certificate
+
+Assign the ticket to the Software team.
+
+
+Initial Setup
+-------------
+
+You will be using the [OSG Build Tools](/software/osg-build-tools) to interact with Koji.
+You can use them on either your own machine or on your UW Comp Sci login node such as `moria`.
+
 
-Using Koji requires:
+### Setting up on moria
 
--   `osg-build` version 1.6.3 or later.
--   `koji` 1.6.0-2.osg or later. **Note** that you want a koji build from osg; the output of `rpm -q koji` should end in ".osg".
+Perform the following to set up the build tools on `moria`:
 
-Both pieces of software are available from the osg repositories. `osg-build` may also be obtained from GitHub by cloning out `https://github.com/opensciencegrid/osg-build`
+1.  Clone the osg-build git repo
 
-#### Special instructions for UW-Madison CSL machines:
+        :::console
+        you@moria$ git clone https://github.com/opensciencegrid/osg-build $HOME/osg-build
 
--   Clone the osg-build GitHub repo:
+1.  Set your `$PATH`:
 
         :::console
-        [you@host]$ git clone https://github.com/opensciencegrid/osg-build
+        you@moria$ export PATH=$PATH:$HOME/osg-build
+        you@moria$ export PATH=$PATH:/p/vdt/workspace/quilt/bin
+        you@moria$ export PATH=$PATH:/p/vdt/workspace/tarball-client/stable/sys
```

Comment:
not that there's anything wrong with this example, but fyi the `exports` are redundant.  Only one `export` is needed to mark a variable name for export (which includes future changes to it). (Additionally `PATH` might be exported by default - but it doesn't hurt to put one in there for clarity.)

Reply:
yeah I just have them in there to line them up.